doc/source/user_guide/advanced.rst

```diff
@@ -402,6 +395,7 @@ slicers on a single axis.
 Furthermore, you can *set* the values using the following methods.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
Is this the pdep warning? Best to just change this example or remove if it won't work anymore

Reply:
doesn't seem to be necessary anymore, lets see what ci has to say

Reply:
I'll keep this in for now, I can't reproduce locally but it shows up on ci, it's weird